internal/cli/atlas/deployments/logs.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deployments
+
+import (
+	"context"
+	"path"
+	"strings"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/deployments/options"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/log"
+	"github.com/mongodb/mongodb-atlas-cli/internal/podman"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+type DownloadOpts struct {
+	cli.OutputOpts
+	cli.GlobalOpts
+	options.DeploymentOpts
+	fs afero.Fs
+}
+
+const fileMode = 0644
+
+func (opts *DownloadOpts) Run(ctx context.Context) error {
+	return opts.RunLocal(ctx)
+}
+
+func (opts *DownloadOpts) RunLocal(ctx context.Context) error {
+	if err := opts.PodmanClient.Ready(ctx); err != nil {
+		return err
+	}
+
+	if err := opts.DetectLocalDeploymentName(ctx); err != nil {
+		return err
+	}
+
+	logs, err := opts.PodmanClient.ContainerLogs(ctx, opts.LocalMongodHostname())
+	if err != nil {
+		return err
+	}
+
+	home, err := config.AtlasCLIConfigHome()
+	if err != nil {
+		return err
+	}
+
+	filepath := path.Join(home, "/", opts.DeploymentName+".log")
+
+	var data = []byte(strings.Join(logs, "\n"))
+	if err := afero.WriteFile(opts.fs, filepath, data, fileMode); err != nil {
+		return err
+	}
+
+	return opts.Print(filepath)
```

Comment:
last time I remember talking about this feature, we were considering printing the output instead of saving it to a file.
Then the user would be able to pipe the content of the logs (save it to a file, tail grep etc). Maybe there was a follow-up conversation and we changed back to file output?
cc @fmenezes

Reply:
sorry maybe I missed this conversation or don't remember, here I tried to follow the design https://www.figma.com/proto/OdPAIJxJFEZ2ZPZ9WOwTcu/Local-UX-for-Atlas-CLI?type=design&node-id=117-747&t=WDVadUELAj6YFc82-0&scaling=min-zoom&page-id=64%3A1654&starting-point-node-id=91%3A1908